stdlib/float.mli

```diff
@@ -217,6 +256,14 @@ external floor : float -> float = "caml_floor_float" "floor"
     equal to [f].
     The result is returned as a float. *)
 
+external nextafter : float -> float -> float
+  = "caml_nextafter_float" "caml_nextafter" [@@unboxed] [@@noalloc]
+(** [nextafter x y] returns the next representable floating-point
+   value following [x] in the direction of [y].  If [y] is less than
+   [x], these functions will return the largest representable number
```

Comment:
"these functions" seems to be from the C documentation...

Reply:
It is – the documentation is heavily inspired from the C one.

Reply:
I meant: shouldn't it be "this function"?

Reply:
I modified it in the commit below.